runtime/kusama/src/lib.rs

```diff
@@ -1484,12 +1484,9 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
 ///
 /// This contains the combined migrations of the last 10 releases. It allows to skip runtime
 /// upgrades in case governance decides to do so. THE ORDER IS IMPORTANT.
-pub type Migrations = (
-	migrations::V0940,
-	migrations::V0941,
-	migrations::V0942,
-	migrations::Unreleased,
-);
+pub type Migrations =
+		(migrations::V0940, migrations::V0941,
+	migrations::V0942, migrations::Unreleased);
```

Comment:
I am no fan of what rustfmt does here.
The list you had before is much more readable.

I would suggest adding `#[rustfmt_skip]` for this `Migrations` type as well as for `Unreleased` and keep one migration / version per line. That also helps with your comment about having this single migration in a release.